### PR TITLE
Newsletter-140:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**

### CODE REVIEW

**Comment by bitschmidty** [comment]
@AkioNak I see 👀  emoji, does that mean reviewed or going to review? :)

**Comment by AkioNak** [comment]
@bitschmidty I am going to review.

File: _posts/ja/newsletters/2021-03-17-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  LNのファンディング・トランザクションは、トランザクションのマリアビリティがある場合には安全ではありません。
+  Segwitはほとんどのトランザクションで懸念される第三者によるマリアビリティを排除しましたが、
+  [Replace-by-Fee][topic rbf] (RBF)を使用してファンディング・トランザクションの手数料を引き上げるといった、
+  トランザクションの作成者自信がそのtxidを変異させるケースには対応していません。
+  txidが変異すると、事前に署名された払い戻しトランザクションが無効になるため、ユーザーは自身の資金を取り戻すことができません。
+  さらにリモートノードは自動的にファンディング・トランザクションを確認できないため、
+  資金提供者が資金を取り戻す支援をできない可能性があります。
+
+    今週、Rusty Russellは、この問題を抱えるユーザーが資金を取り戻せるよう、
+    C-Lightningに実装した迅速かつ実験的な機能についてLightning-Devメーリングリストに[投稿しました][russell funding rescue]。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-03-17-newsletter.md:25]: I'm not sure, but I think `quick' is not the feature itself, but for its implementation period ( just a day from issue opened).

```suggestion
    C-Lightningに素早く実装した実験的な機能についてLightning-Devメーリングリストに[投稿しました][russell funding rescue]。
```

File: _posts/ja/newsletters/2021-03-17-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- [HWI 2.0.0][hwi 2.0.0]は、HWIの次のメジャーバージョンのリリースです。
+  とりわけ、BitBox02でのマルチシグのサポートやドキュメントの改善、
+  Trezorでの`OP_RETURN`アウトプットの支払いのサポートなどが含まれています。
+
+- [Rust-Lightning 0.0.13][]は、このLNライブラリの最新リリースで、
+  [マルチパス・ペイメント][topic multipath payments]との前方互換性や
+  [Taproot][topic taproot]などの将来のスクリプト・アップグレードを目的とした改善が含まれています。
+
+- [BTCPay Server 1.0.7.0][]は、このセルフホスト型の支払い処理ソフトウェアの最新リリースです。
+  注目すべき改善点は、より機能的で視覚的に魅力的なウォレットセットアップウィザードや、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-03-17-newsletter.md:48]: (visually) appealing -> s/魅力的な/分かりやすい/

```suggestion
  注目すべき改善点は、より機能的で視覚的に分かりやすいウォレットセットアップウィザードや、
```

File: _posts/ja/newsletters/2021-03-17-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+*今週の[Bitcoin Core][bitcoin core repo]、
+[C-Lightning][c-lightning repo]、[Eclair][eclair repo]、[LND][lnd repo]、
+[Rust-Lightning][rust-lightning repo]、[libsecp256k1][libsecp256k1 repo]、
+[Hardware Wallet Interface (HWI)][hwi repo]、
+[Rust Bitcoin][rust bitcoin repo]、[BTCPay Server][btcpay server repo]、
+[Bitcoin Improvement Proposals（BIP）][bips repo]、および
+[Lightning BOLTs][bolts repo]の注目すべき変更点。*
+
+- [Bitcoin Core #21007][]では、新しい設定オプション`-daemonwait`が追加されました。
+  Bitcoin Coreは、初期バージョンから`-daemon`設定オプションでプログラムを起動することで、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-03-17-newsletter.md:62]: s/初期バージョン/初期の頃のバージョン/
s/設定オプションで/設定オプションを指定して/

```suggestion
  Bitcoin Coreは、初期の頃のバージョンから`-daemon`設定オプションを指定してプログラムを起動することで、
```

File: _posts/ja/newsletters/2021-03-17-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  `-daemon`オプションを指定すると、プログラムはすぐにバックグランドでデーモンプロセスを開始します。
+  新しい`-daemonwait`オプションも同様ですが、初期化の完了後にデーモンプロセスをバックグランドで起動します。
+  これにより、ユーザーや親プロセスは、プログラムの出力や終了コードを観察することで、
+  デーモンの起動が成功したかどうかをより簡単に知ることができます。
+
+- [C-Lightning #4404][]では、`keysend` RPC（[ニュースレター #107][news107 keysend]参照）
+  がこの機能をサポートしていることを明示的に通知していないノードにもメッセージを送信できるようになりました。
+  [議論されている][fromknecht keysend]ように、通知は標準化されておらず、
+  LNDで実装されている手順は通知に依存していないため、
+  この変更によりC-LightningはLNDが対応できるノードとほぼ同じセットにメッセージを送信できるようになります。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-03-17-newsletter.md:73]: I think to repeat using 'ノード' is more natural in Japanese.
s/セット/ノード/

```suggestion
  この変更によりC-LightningはLNDが対応できるノードとほぼ同じノードにメッセージを送信できるようになります。
```

File: _posts/ja/newsletters/2021-03-17-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  最も注目すべき点は、少なくとも一時的にProof of Discrete Log Equivalency (PODLE)の使用をやめたことです
+  （PODLEのオリジナルの議論は[ニュースレター #83][news83 podle]を、
+  代替手段についての議論は[ニュースレター #131][news131 podle]を参照）。
+  このマージに続いて、特別なビルドフラグでC-Lightningをコンパイルする必要なく（ただし、特別な設定オプションは必要）、
+  デュアル・ファンディングの実験がより利用しやすくなる[新しいPR][c-lightning #4427]が公開されました。
+
+- [LND #5083][]では、[PSBT][topic psbt]を標準入力（stdin）のファイルディスクリプタからではなく、
+  ファイルから読み込めるようになりました。
+  一部の端末では、標準入力に同時に追加（貼り付けなど）できる文字数に[制限][lnd #5080]があり、
+  4096 base64文字（3,072バイトのバイナリに相当）を超えるPSBTは使用できませんでした。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-03-17-newsletter.md:86]: s/4096 base64文字/base64形式で4096文字/

```suggestion
  base64形式で4096文字（3,072バイトのバイナリに相当）を超えるPSBTは使用できませんでした。
```

**Comment by azuchi** [comment]
@AkioNak Thanks! I fixed.

**Comment by bitschmidty** [comment]
ACK 6e2c4fd97c2ec80c309febd1adafd7c9f3081171

Thanks @azuchi and @AkioNak ! These translations are much appreciated.

**Pull request merged**